testdata/p4_16_samples/advance_ebpf.stf

```diff
@@ -0,0 +1,14 @@
+packet 0 08 00 01
```

Comment:
all these tests are positive, can you add a negative one, where a packet is dropped?

Reply:
Since `advance` works only with constant arguments I changed a testcase that checks only constant + added negative one